crates/nu-protocol/src/engine/state_delta.rs

```diff
@@ -2,7 +2,6 @@ use super::{usage::Usage, Command, EngineState, OverlayFrame, ScopeFrame, Variab
 use crate::ast::Block;
 use crate::Module;
 
-#[cfg(feature = "plugin")]
```

Comment:
Oh, that went under the radar...

mental note: `--no-default-features` for the often dreamt of nightly extended test suite.